charts/kargo/templates/api/configmap.yaml

```diff
@@ -60,4 +60,10 @@ data:
   ARGOCD_URLS: {{ range $key, $val := .Values.api.argocd.urls }}{{ $key }}={{ $val }},{{- end }}
   {{- end }}
   ROLLOUTS_INTEGRATION_ENABLED: {{ quote .Values.api.rollouts.integrationEnabled }}
+  {{- if .Values.controller.rollouts.integrationEnabled }}
```

Comment:
It bugs me that we're using a `.Values.controller...` setting for an API server concern...

I suspect we may eventually remove support for two separate clients for Kargo control plane and Rollouts control plane, so I wouldn't waste time fixing this. Just calling out that I recognize the peculiarity of this and think we can let it slide for now.

Reply:
Actually, I take that back. It's easy enough to fix this. Sorry for the flip-flopping.

Reply:
Also, we'll still need this flag even if we remove support for the second client, so ya... any initial thought I had that this was temporary was completely off.

Reply:
Is there a scenario in which API and controller are configured differently? If not, we could move `.rollouts` up under `.Values` itself, with fallback to the previous fields where applicable.

Reply:
Addressed the remaining `.controller` reference by amending c885b167f1f660f1f194422a42d7eba068b75df3

Reply:
> Is there a scenario in which API and controller are configured differently?

In a single Helm release... no.

API server is only ever installed on the control plane. Controllers can be more distributed, but any API server and controller that are co-located would probably agree on what integrations are enabled.

I agree we could move it up, although it is a breaking change... 🤔